lib/hourly_logger_rotator/logs_lifetime.rb

```diff
@@ -0,0 +1,19 @@
+class HourlyLoggerRotator::LogsLifetime
+  attr_accessor :dirname
+
+  def initialize(dirname)
+    self.dirname = dirname
+  end
+
+  def call
+    return unless HourlyLoggerRotator.logs_lifetime
+    Pathname.new(dirname).each_child do |entry|
+      next unless entry.file?
+
+      if entry.basename.to_s =~ /\.log\.\d+/ && entry.mtime < HourlyLoggerRotator.logs_lifetime.ago
```

Comment:
I think it's better to add `\z` at the end of this regexp.

Reply:
why?